python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -102,10 +102,13 @@ def from_pretrained(cls,
     @classmethod
     def load_convert(cls, q_k, optimize_model, *args, **kwargs):
         from .convert import ggml_convert_low_bit
-        invalidInputError(q_k in ggml_tensor_qtype,
+        invalidInputError(q_k in ggml_tensor_qtype or q_k == "fp16",
```

Comment:
add "fp16" to `ggml_tensor_qtype`

Reply:
Have added "fp16" to the end of `ggml_tensor_type`.